css/base.css

```diff
@@ -7,7 +7,7 @@ header{
     color: goldenrod;
 }
 h1{
-    line-height: 55px;
+    line-height: 100px;
```

Comment:
line height

Reply:
Would you like me to explain more about this?